### PR TITLE
Login page draft

### PR DESCRIPTION
**Pull request by egurung11**
this is the draft for the login/register page, no logic - will need help connecting it with stuff

### CODE REVIEW

**Comment by egurung11** [comment]
image

^ does not look good, but is a base to start with, will change

File: BAR/Components/Pages/Login.razor [hunk 1]
```diff
@@ -0,0 +1,105 @@
+﻿@page "/loginregister"
+@layout BAR.Components.Layout.LandingLayout
+@rendermode InteractiveServer
+
+<style>
+
```
Comment (max-dejesus) [inline BAR/Components/Pages/Login.razor]: adding your own style is cool! however id recommend looking at the bootstrap docs to see what they offer for auto layouts and stuff, honestly makes it pretty simple

https://getbootstrap.com/docs/4.1/getting-started/introduction/

File: BAR/Components/Pages/Login.razor [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            <label for="usn"><b>Username: </b></label>
+            <input type="username" placeholder="Username" name="usn" id="usn" required>
+
+            <label for="psw"><b>Password: </b></label>
+            <input type="password" placeholder="Password" name="psw" id="psw" required>
+            
+            <hr>
+            <p>By creating an account you agree to our <a href="#">Terms & Privacy</a>.</p>
+
+            <button type="submit" class="loginbtn">Login</button>
```
Comment (max-dejesus) [inline BAR/Components/Pages/Login.razor]: forgot password button/link should be added here somewhere

**Review by max-dejesus — CHANGES_REQUESTED** [review]
o7

File: BAR/Components/Pages/Login.razor (hunk 1 above)
Comment (max-dejesus) [reply BAR/Components/Pages/Login.razor]: additionally you can make css files titled after your razor file and itll pull in the css automatically (i.e. Landing.razor can have Landing.razor.css as its stylesheet)

**Review by max-dejesus — APPROVED** [review]
ok

**Review by haley-v — APPROVED** [review]
cool beans 👍

**Pull request merged**